internal/cmdtest/jimmsuite.go

```diff
@@ -150,6 +154,17 @@ func (s *JimmCmdSuite) AddAdminUser(c *gc.C, email string) {
 	c.Assert(err, gc.IsNil)
 }
 
+// SetupCLIAccess should be run at the start of all CLI tests that want to communicate with JIMM.
+// It will ensure a user is created in JIMM's in-memory accounts store that matches the username passed in.
+// This is necessary as Juju's CLI library validates the logged-in user matches the expected user from the accounts store.
+// This function will return a login provider that can be used to handle authentication.
+func (s *JimmCmdSuite) SetupCLIAccess(c *gc.C, username string) api.LoginProvider {
```

Comment:
I agree, and that's my bad. I think I'd initially kept these changes separated into a specific commit but there are now a lot of commits.
I made the change because the CLI tests were broken when I upgraded the Juju dependency.